parsers/US_SPP.py

```diff
@@ -380,50 +380,56 @@ def format_exchange_data(
     data_filtered = data.loc[data["zone_key2"] == zone_key2]
     data_filtered = data_filtered.groupby([data_filtered.index])["exchange"].sum()
 
-    all_data_points = []
+    exchange_list = ExchangeList(logger)
     for dt in data_filtered.index:
         data_dt = data_filtered.loc[data_filtered.index == dt]
-        data_point = {
-            "sortedZoneKeys": sorted_zone_keys,
-            "netFlow": round(data_dt.values[0], 4),
-            "datetime": dt.to_pydatetime(),
-            "source": "spp.org",
-        }
-        all_data_points.append(data_point)
-    validated_data_points = [x for x in all_data_points if validate_exchange(x, logger)]
+        exchange_list.append(
+            zoneKey=sorted_zone_keys,
```

Comment:
`sorted_zone_keys` need to be wrapped with a ZoneKey instance for the type to match.